openfl/federated/plan/plan.py

```diff
@@ -420,8 +443,13 @@ def get_core_task_runner(self, data_loader=None,
 
         return self.runner_
 
-    def get_collaborator(self, collaborator_name, root_certificate=None, private_key=None,
-                         certificate=None, task_runner=None, client=None, shard_descriptor=None):
+    def get_collaborator(self, collaborator_name: str,
+                         root_certificate: Union[Path, str, None] = None,
+                         private_key: Union[Path, str, None] = None,
+                         certificate: Union[Path, str, None] = None,
```

Comment:
```suggestion
                         root_certificate: Optional[Path, str] = None,
                         private_key: Optional[Path, str] = None,
                         certificate: Optional[Path, str] = None,
```

Reply:
`Optional`  take's only one argument as I understand, it will fall with the error `TypeError: Optional[t] requires a single type`